src/repl.rs

```diff
@@ -105,10 +106,15 @@ impl State {
     pub async fn set_idle_transaction_timeout(&mut self) -> anyhow::Result<()> {
         if let Some(conn) = &mut self.connection {
             if conn.protocol().is_at_least(0, 13) {
-                conn.execute(
-                    "CONFIGURE SESSION SET \
-                 session_idle_transaction_timeout := <std::duration>'5m';"
-                ).await.context(
+                let secs = self.idle_transaction_timeout.unwrap_or(0);
+                log::info!(
+                    "Setting session_idle_transaction_timeout to {}s", secs
+                );
+                conn.execute(&format!(
+                    "CONFIGURE SESSION SET session_idle_transaction_timeout \
+                     := <std::duration>'{}s'",
```

Comment:
You don't need 's' suffix; numbers without a unit are assumed to be seconds for std::duration.

```suggestion
                     := <std::duration>'{}'",
```